tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Shared/SpanPointers.cs

```diff
@@ -79,6 +116,22 @@ internal static string ConcatenateComponents(string bucketName, string key, stri
         return StringBuilderCache.GetStringAndRelease(builder);
     }
 
+    internal static string ConcatenateComponents(string tableName, string key1, string value1, string key2, string value2)
+    {
+        var builder = StringBuilderCache.Acquire();
+        builder.Append(tableName);
+        builder.Append('|');
+        builder.Append(key1);
+        builder.Append('|');
+        builder.Append(value1);
+        builder.Append('|');
+        builder.Append(key2);
+        builder.Append('|');
+        builder.Append(value2);
+
+        return StringBuilderCache.GetStringAndRelease(builder);
```

Comment:
Since this concatenation will always happen the same way (no conditions or loops or other code branches), you can you simple concatenation:

```suggestion
        return $"{tableName}|{key1}|{value1}|{key2}|{value2}";
```